src/frontend/src/lib/components/modals/create-post/CreatePostModal.svelte

```diff
@@ -92,7 +91,7 @@
 {#if $modalStore[0]}
   <div class="card p-4 w-modal shadow-xl space-y-4">
     <header class="text-2xl font-bold">{$modalStore[0].title}</header>
-    <article>{$modalStore[0].body}</article>
+    <!--    <article>{$modalStore[0].body}</article>-->
```

Comment:
Can we remove this or do we have to keep there this commented out code?

Reply:
Need to keep it there for now

Reply:
Alright then! I've tested it locally, feel free to merge it. Then we will check how it behaves on dev environment.